omnisafe/algorithms/algo_wrapper.py

```diff
@@ -144,7 +144,8 @@ def _init_checks(self) -> None:
         assert self.cfgs.train_cfgs.parallel > 0, 'parallel must be greater than 0!'
         assert (
             self.env_id in support_envs()
-        ), f"{self.env_id} doesn't exist. Please choose from {support_envs()}."
+        ), f"{self.env_id} doesn't exist. Please choose from {support_envs()}.\
+            \nIf you are using Safe Isaac Gym environments, please install Isaac Gym first."
```

Comment:
Maybe move this into safety_isaac_gym_env will be better.

Reply:
The current logic is that when trying to import the Isaac Gym package, there will be no errors or warnings. This is to avoid confusion for users who do not use Safe Isaac Gym. When users attempt to use the Safe Isaac Gym environment without the package installed, they will then see this message.